### DISCUSSION TITLE
Debugging ruby code in devContainer with `ruby-debug-ide` - Missing open ports?

### DISCUSSION THREAD

**Discussion by N7K4** [devcontainers/spec#240] [Q&A]
Hi I am a ruby developer, doing some jekyll plugins.

I have moved my ruby dev environment to a devContainer in VScode (Win10, WSL2 Ubuntu). Ruby v3.1 with minitest as test frameworks, sometimes also rspec. But here I am talking just about minitest. Lauch.json and Settings.json from vsCode are setup and working.

When I am working on WSL2 with Ubuntu in vsCode I can run and DBEUG tests incl. hiting breakpoints.  

When I am inside the DevContainer, I can run the tests fine (get green and red respective to the tests). But debugging are not working. I see a short reaction and then the state of the test is updated, but when I have breakpoints they will not be reached and the program will not pause there. 

I tried to forward ports in my devcontainer.json. But nothing changed. 

Any idea, how I can debug unit tests with minitest inside a ruby devcontainer? Many thanks in advance.

**Comment by bamurtaugh** [comment]
@alexander-smolyakov if you get a chance, could you check this out? Thanks so much!
